packages/web/src/js/Collapse.ts

```diff
@@ -59,14 +61,32 @@ class Collapse extends BaseComponent {
     return `${this.NAME}`;
   }
 
+  siblingsControlHandler(trigger: HTMLElement) {
+    if (this.target?.dataset.parent) {
+      const parentElement = selectorEngine.findOne(`#${this.target?.dataset.parent}`);
```

Comment:
```suggestion
      const parentElement = selectorEngine.findOne(this.target?.dataset.parent);
```

Do not limit `data-parent` to id only. It is up to developer what parent will be using.